js/packages/web/src/views/home/purchaseArt.tsx

```diff
@@ -86,25 +86,26 @@ const updateTokenSane = async (
         TOKEN_SALE_MASTER_ACCOUNT_ADDRESS,
       );
       setAccountFn(masterAccount as MasterAccount);
-      if (account && account.numTokens.toNumber() > 0) {
+      if (account && account.allocated.toNumber() > 0) {
         // set progress value and amount remaining
-        if (account.counter.toNumber() <= 1) {
+
+        if (account.sold.toNumber() <= 1) {
           setProgressValueFn(0);
-          setAmountRemainingFn(account.numTokens.toNumber());
+          setAmountRemainingFn(account.allocated.toNumber());
         } else {
           setProgressValueFn(
             Math.round(
-              (account.counter.toNumber() / account.numTokens.toNumber()) * 100,
+              (account.sold.toNumber() / account.allocated.toNumber()) * 100,
             ),
           );
           setAmountRemainingFn(
-            account.numTokens.toNumber() - account.counter.toNumber(),
+            account.allocated.toNumber() - account.sold.toNumber(),
           );
         }
         // set price
-        if (account.counter.toNumber() >= 2222) {
+        if (account.sold.toNumber() >= 2222) {
```

Comment:
I think this should be `1421` to match smart contract https://github.com/thuglabs/simple-token-sale/blob/v2-zerocopy/programs/simple-token-sale/src/lib.rs#L297